src/components/Check-In/index.tsx

```diff
@@ -1,14 +1,24 @@
-import React, { useState, BaseSyntheticEvent } from 'react';
+import React, { useState, useEffect, BaseSyntheticEvent } from 'react';
 import { VStack, Heading, Text, Button, Input, Box } from '@chakra-ui/react';
 import { useQuery } from 'react-query';
 
 import axiosInstance from '../../api';
 import { toastError, toastSuccess } from '../../utils/toast';
+import Confetti from 'react-confetti';
 
 const CheckIn = (): React.ReactElement => {
+  const [isSubmitted, setIsSubmitted] = useState(false);
   const [eventKey, setEventKey] = useState('');
   const [eventKeyError, setEventKeyError] = useState(false);
 
+  useEffect(() => {
+    const storedIsSubmitted = localStorage.getItem('isSubmitted');
+    if (storedIsSubmitted === 'true') {
+      setIsSubmitted(true);
+      localStorage.removeItem('isSubmitted');
+    }
+  }, []);
+
```

Comment:
instead of reloading the page & saving to local storage, we can use a timeout & `useState`! [this](https://github.com/alampros/react-confetti/issues/92) should be a good starting point